<launch>
  <node type ="laser2pc_node" pkg="bag_tools" name="laser2pc" output="screen" >
    <param name="pc_frame" value="map" />
    <param name="scan_topic" value="/scan" />
  </node>
</launch>
